### PR TITLE
Gestio Re-balance

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- Rechambered gestio rifle to .30.
- Slightly lowered spread (10 -> 5) for wielded state.
- Fixed visual bug with default empty magazine not showing fill states correctly.
- Moved files around a bit.

## Why / Balance
1. We don't have weapons chambered in .30 in Liberation Station.
2. Gestio rifle lags behind even if compared to Argenty revolver.

## How to test
1. Spawn Liberation Station, lecter rifle and akms.
2. Buy Gestio rifle, empty low cap magazine and boxes of .30 and .20 cartridges.
3. Fail to load empty low cap mags with .20 ammo, succeed loading low cap mags with .30 ammo.
4. Low cap mags should be compatible with akms, and not compatible with lecter rifle.

## Media
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- tweak: Changed Gestio to use .30 ammo, reduced spread a bit when the rifle is wielded.

### CODE REVIEW

**Comment by ErhardSteinhauer** [comment]
@Qulibly notifying you of proposed changes to your rifle.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml [hunk 1]
```diff
@@ -30,27 +30,27 @@
   - type: Wieldable
   - type: GunWieldBonus
     minAngle: -2
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml:32]: Minimum angle should be 1 degree.  If the Mosin has a minimum angle of 1 degree, this should too.
```suggestion
    minAngle: -1
```

Also, if this is a "marksman's rifle", why is the _unwielded_ minimum angle 2, making this most suitable for unwielded run-and-gun pot shots?  Maybe this was an oversight, but I do think it should use the (boring) defaults of 21 and -20.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   name: Gestio
   parent: BaseWeaponRifle
   id: WeaponRifleGestio
-  description: An old prototype burst-fire NanoTrasen marksman rifle. Manufactured by Silver Industries. It hasn't ever left the trials. Given it's antiquity it is considered a civilian grade weapon.   Uses .20 rifle ammo.
+  description: An old prototype burst-fire NanoTrasen marksman rifle. Manufactured by Silver Industries. It hasn't ever left the trials. Given it's antiquity it is considered a civilian grade weapon.   Uses .30 rifle ammo.
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml:5]: French spacing.
```suggestion
  description: An old prototype burst-fire NanoTrasen marksman rifle. Manufactured by Silver Industries. It hasn't ever left the trials. Given it's antiquity it is considered a civilian grade weapon.  Uses .30 rifle ammo.
```

**Review by whatston3 — CHANGES_REQUESTED** [review]
I'm not really enthused about these changes.  This seems to completely invalidate any value the SVT has - same caliber, higher capacity, higher rate of fire, and (at the moment) amazing unwielded capabilities.  Burst fire could be more interesting if you had an extreme rate of fire, and guaranteed three shots with a larger window, and would leave a role for the SVT as a mid-tier semi-auto rifle.

I left a few comments for changes I thought were useful.  Curious to see what you think.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml (hunk 2 above)
Comment (arimah) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml:5]: While you're at it, fix the grammar too: "Given <ins>its</ins> antiquity"

**Comment by ErhardSteinhauer** [comment]
> amazing unwielded capabilities.

Yeah, gotta fix that actually.

> guaranteed three shots with a larger window

I'm not sure it's possible (without extensive c# coding) to increase the pause between bursts.

**Review by dvir001 — APPROVED** [review]

**Pull request merged**